src/arcade/eng/common/core-templates/steps/source-build.yml

```diff
@@ -80,26 +66,12 @@ steps:
       /p:DotNetBuildRepo=true \
   displayName: Build
 
-# Upload build logs for diagnosis.
-- task: CopyFiles@2
-  displayName: Prepare BuildLogs staging directory
-  inputs:
-    SourceFolder: '$(Build.SourcesDirectory)'
-    Contents: |
-      **/*.log
-      **/*.binlog
-      artifacts/sb/prebuilt-report/**
-    TargetFolder: '$(Build.StagingDirectory)/BuildLogs'
-    CleanTargetFolder: true
-  continueOnError: true
-  condition: succeededOrFailed()
-
 - template: /eng/common/core-templates/steps/publish-pipeline-artifacts.yml
   parameters:
     is1ESPipeline: ${{ parameters.is1ESPipeline }}
     args:
       displayName: Publish BuildLogs
-      targetPath: '$(Build.StagingDirectory)/BuildLogs'
+      targetPath: artifacts/log/$[ coalesce(variables._BuildConfig, 'Release') ]/**
```

Comment:
the ** here looks suspect, the AzDO publish task doesn't support wildcards afaik

Reply:
Oh right, thanks for spotting. I will remove the `/**` part. DIrectories are supported, according to https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/publish-pipeline-artifact-v1?view=azure-pipelines#inputs